operate/services/manage.py

```diff
@@ -54,7 +54,7 @@
 )
 from operate.data.contracts.staking_token.contract import StakingTokenContract
 from operate.keys import KeysManager
-from operate.ledger import PUBLIC_RPCS, get_currency_denom
+from operate.ledger import get_currency_denom, get_default_rpc
```

Comment:
For consistency, avoided use of `PUBLIC_RPCS` (now removed) and `DEFAULT_RPCS` dictionaries, and used this already existing method `get_default_rpc`.